en/concepts/scene/fog.md

```diff
@@ -0,0 +1,74 @@
+## Global fog type
+
+Four types of global fog are provided inside the engine:
+
+1. Linear Fog
+2. Exponential Fog
+3. Exponential Squared Fog
+4. Layered Fog
+
+The global fog function is used to obtain the fogging mixing factor through the camera and the model vertices, to get different global fog effects.
```

Comment:
The global fog function is used to obtain different fog effects with the fogging mixing factor affecting globally the model vertices.